compiler/compiler.go

```diff
@@ -405,6 +406,9 @@ func (c *Compiler) Compile(mainPath string) []error {
 	// Tell the optimizer that runtime.alloc is an allocator, meaning that it
 	// returns values that are never null and never alias to an existing value.
 	for _, attrName := range []string{"noalias", "nonnull"} {
+		if c.mod.NamedFunction("runtime.alloc").IsNil() {
+			panic("no runtime.alloc")
```

Comment:
This check whether a function exists (and similar checks below) are unrelated to rpi3 support.